style: add Pull Request template (#535)
lukso-network/lsp-smart-contracts: .github/PULL_REQUEST_TEMPLATE.md

<!-- Thank you for your interest in contributing to LUKSO! -->

<!-- Consider opening an issue for discussion prior to submitting a PR. -->
<!-- Consider checking CONTRIBUTING.md before contributing. -->

> <!-- New features will be merged faster if they were first discussed and designed with the team. -->

Reviewer: Maybe we can suggest to either:
- open an issue.
- go to the discussion tab of the repo?

first?

<img width="182" alt="image" src="https://user-images.githubusercontent.com/31145285/228162878-94c04cea-bd9b-4f56-bd99-c3e9e8a0a15d.png">

# What does this PR introduce?

<!-- Keep the sub-header that suits the PR and remove the rest -->

<!-- Changes that potentially causes other components to fail (changes in interfaceIds, function signatures, behavior, etc ..) --->
<!---
## ⚠️ BREAKING CHANGES
---->

<!---
## 🚀 Feature
---->

<!---
## 🐛 Bug
---->

<!---
## ♻️ Refactor
---->

<!---
## 🧪 Tests
---->

<!---
## ⚡️ Performance
---->

<!---
## 🎨 Style
---->

<!---
## 📄 Documentation
---->

<!---
## 📦 Build
---->

<!---
## 🤖 CI
---->

Fixes #???? <!-- Fill in with issue number -->

<!-- Describe the changes introduced in this pull request here. -->

<!-- Include any context necessary for understanding the PR's purpose. (Images, links, etc ..) -->

### PR Checklist

> <!-- Before merging the pull request all of the following must be complete. -->

Reviewer: ```suggestion
<!-- Before merging the pull request, making sure you have run locally the following. -->
```

<!-- Feel free to submit a PR or Draft PR even if some items are pending. -->

> <!-- Some of the items may not apply. -->

Reviewer: ```suggestion
<!-- (Some of the items may not apply.) -->
```

> - [ ] Tests

Reviewer: Is it what was meant here?

```suggestion
- [ ] Wrote Tests
```

- [ ] Documentation
- [ ] Ran `npm run test`
- [ ] Ran `npm run linter`

> - [ ] Ran `npm run prettier`

Reviewer: I suggest we put them in the same order as the CI:

- [ ] Ran `npm run linter`
- [ ] Ran `npm run prettier`
- [ ] Ran `npm run build`
- [ ] Ran `npm run test`

<img width="290" alt="image" src="https://user-images.githubusercontent.com/31145285/228164221-fd7e007b-f7db-4390-b0d8-eb89c19228f8.png">
